docs/yul.rst

```diff
@@ -1076,6 +1076,23 @@ regular strings in native encoding. For code,
 
 Above, ``Block`` refers to ``Block`` in the Yul code grammar explained in the previous chapter.
 
+.. note::
```

Comment:
Added this.

Reply:
Wonder if this is a good reason to go for `@` as the prefix? (As proposed as an option in the issue.)

Reply:
If the fields are meant to be inaccessible, then `.` is better. If they are meant to be accessible then `@` is better.